doc/manuscript.Rmd

```diff
@@ -72,28 +72,206 @@ pretty <- function(x) format(x, big.mark = ",")
+```
+
+## The effects of sleep onset on physical activity
+
+We estimated the effect of sleep onset on physical activity by age. Results, controlling for sex, SES, and BMI are presented in Table \@ref(tab:PA-by-sleep-onset) and Figure \@ref(fig:PA-by-sleep-onset-fig). There were strong U-shaped relationships where average sleep onset was linked to the highest levels of physical activity volume and intensity. The U-shaped relationship between sleep onset and physical activity volume attenuated for older participants.
+
+```{r PA-by-sleep-onset}
+var_names <- c(
+  "scale_pa_volume by scale_sleep_onset_lag",
+  "scale_pa_intensity by scale_sleep_onset_lag"
+)
+
+PA_by_sleep_onset <- lapply(model_list[var_names], function(x){
+tab <- format_table(x$table)  
+tab
+})
+
+names(PA_by_sleep_onset) <- gsub("scale_", "", names(PA_by_sleep_onset)) |> 
+  gsub("\\sby.*", "", x = _ ) |> 
+  gsub("_", " ", x = _ ) |> 
+  fix_model_names()
+
+  papaja::apa_table(PA_by_sleep_onset,
+                    caption = "Physical activity by sleep onset controlling for SES, gender and BMI",
+                    note = note,
+                    escape = FALSE)
+```
+
+```{r PA-by-sleep-onset-fig, fig.cap = "Physical activty by sleep onset"}
+knitr::include_graphics(here::here("Figures/PA on sleep_onset_lag.jpg"), error = FALSE)
+```
+
+
+## The effects of sleep regularity on physical activity
+
+We estimated the effect of sleep regularity on physical activity by age. Results, controlling for sex, SES, and BMI are presented in Table \@ref(tab:PA-by-sleep-regularity) and Figure \@ref(fig:PA-by-sleep-regularity-fig). There was a U-shaped relationship between sleep regularity and physical activity volume. Participants with below-average sleep regularity tended to have average physical activity volume. Increases in regularity above the average were linked to greater physical activity volume. There was a strong linear relationship between sleep regularity and physical activity intensity which slightly attenuated with age. Greater sleep regularity was associated with greater physical activity the following day.
+
+```{r PA-by-sleep-regularity}
+var_names <- c(
+  "scale_pa_volume by scale_sleep_regularity_lag",
+  "scale_pa_intensity by scale_sleep_regularity_lag"
+)
+
+PA_by_sleep_regularity <- lapply(model_list[var_names], function(x){
+tab <- format_table(x$table)  
+tab
+})
+
+names(PA_by_sleep_regularity) <- gsub("scale_", "", names(PA_by_sleep_regularity)) |> 
+  gsub("\\sby.*", "", x = _ ) |> 
+  gsub("_", " ", x = _ ) |> 
+  fix_model_names()
+
+  papaja::apa_table(PA_by_sleep_regularity,
+                    caption = "Physical activity by sleep regularity controlling for SES, gender and BMI",
+                    note = note,
+                    escape = FALSE)
+```
+
+```{r PA-by-sleep-regularity-fig, fig.cap = "Physical activty by sleep regularity"}
+knitr::include_graphics(here::here("Figures/PA on sleep_regularity_lag.jpg"), error = FALSE)
```

Comment:
Sorry one more thought - should there be a dependency for purdy_pictures?

Reply:
I can't see an argument in tar_render to add dependencies. Do we need to do that? Because it also depends on model_list

Reply:
Ahh nvm:
> The R Markdown source should mention dependency targets with tar_load() and tar_read() in the active code chunks (which also allows you to render the report outside the pipeline if the ⁠_targets/⁠ data store already exists)

Reply:
When you put `tar_load` into the Rmd it creates a dependency.